### PR TITLE
[Kotlin Migration][Follow-Up] Time

### PR DESCRIPTION
**Pull request by codingtosh**
## Purpose / Description : 
This is a FollowUp cleanup of Time.java migration to Time.kt

1.  Remove Wildcard import.
2.  Resolve import conflict -> Use import alias.
3.  Resolve Warning -> Kotlin's max() instead of Math.max()

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/libanki/utils/Time.kt [hunk 1]
```diff
@@ -65,8 +65,8 @@ abstract class Time {
     /** Return the first safe ID to use.  */
     fun maxID(db: DB): Long {
         var now = intTimeMS()
-        now = Math.max(now, db.queryLongScalar("SELECT MAX(id) FROM cards"))
-        now = Math.max(now, db.queryLongScalar("SELECT MAX(id) FROM notes"))
+        now = now.coerceAtLeast(db.queryLongScalar("SELECT MAX(id) FROM cards"))
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/libanki/utils/Time.kt:68]: Is there a reason we use this over Kotlin's `max()`

**Review by david-allison-1 — COMMENTED** [review]
One query, but looks good

File: AnkiDroid/src/main/java/com/ichi2/libanki/utils/Time.kt (hunk 1 above)
Comment (codingtosh) [reply AnkiDroid/src/main/java/com/ichi2/libanki/utils/Time.kt:68]: I tried look up the documentation but it wasn't very helpful in elucidating exactly why Android Studio insisted on coerceAtLeast as the top suggestion despite the max() and maxOf() functions. I personally think max() or maxOf would be much better for readability, but I'll defer it to your best judgement on which one to use. 

I could find [this ](https://discuss.kotlinlang.org/t/minof-and-maxof-versus-coerceatleast-and-coerceatmost/2258)discussion, though I'm not sure I understand the nuance still.

Comment (david-allison-1) [reply AnkiDroid/src/main/java/com/ichi2/libanki/utils/Time.kt:68]: Android Studio suggests both:

image

Let's go with `max`, I understand it 😉

**Review by david-allison-1 — APPROVED** [review]
Great! Thanks

**Review by mikehardy — APPROVED** [review]

**Pull request merged**